flixel/tile/FlxTilemap.hx

```diff
@@ -33,6 +33,46 @@ import Std.is as isOfType;
 
 using flixel.util.FlxColorTransformUtil;
 
+#if html5
```

Comment:
Might as well use this approach on all targets?

Reply:
html5 is the only one that loads the bitmaps asynchronously so there's no need for this on other targets. consequently, only html5 has the `onLoad` arg, so we would need compile conditions for 2 versions of the constructor, anyway.

Reply:
Ah, I didn't realize there's another difference, that's unfortunate...